lib/quby/questionnaires/entities/fields.rb

```diff
@@ -21,30 +21,22 @@ class Fields
         # targeted by :depends_on relations.
         attr_reader :input_keys
 
-        def initialize
+        def initialize(questionnaire)
```

Comment:
Netter om een option `check_key_clashes: true/false` mee te geven als dat het enige gebruikte attribuut van de questionnaire is.